src/sections/dataset/dataset-files/files-table/file-actions/file-actions-cell/file-action-buttons/file-options-menu/FileOptionsMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import { File } from '../../../../../../../../files/domain/models/File'
+import { DropdownButton, DropdownHeader, Tooltip } from '@iqss/dataverse-design-system'
+import { PencilFill, ThreeDotsVertical } from 'react-bootstrap-icons'
+import { useSession } from '../../../../../../../session/SessionContext'
+import { EditFilesOptions } from '../../../edit-files-menu/EditFilesOptions'
+import { useTranslation } from 'react-i18next'
+
+export function FileOptionsMenu({ file }: { file: File }) {
+  const { t } = useTranslation('files')
+  const { user } = useSession()
+  const userHasDatasetUpdatePermissions = true // TODO - Implement permissions
+  const datasetHasValidTermsOfAccess = true // TODO - Implement terms of access validation
+  const datasetLockedFromEdits = false // TODO - Ask Guillermo if this a dataset property coming from the api
+  // const isDeleted = false // TODO - Ask Guillermo if this is a file property coming from the api
```

Comment:
It is not currently exposed via API, but can be easily added to the current file endpoint.

Why is it commented?

Reply:
I added a commit with the isDeleted property